### PR TITLE
[element 2.0] Minor updates on docs for Element v2.0

### PR DESCRIPTION
**Pull request by terryytran**

### CODE REVIEW

File: packages/docs/docs/guides/CLI.md [hunk 1]
```diff
@@ -38,12 +39,22 @@ Init a test script, a default configuration file and a minimal environment to ge
 element generate <file>
 ```
 
+**Positionals**
+
+- `file` (string) the test script file name to create. Specify a TypeScript file format, for example, `my-test-script.ts`.
```
Comment (HieuLeKMS) [inline packages/docs/docs/guides/CLI.md:44]: **"Specify a TypeScript file format"** doesn't sound right. Here's my suggestion for your consideration
```suggestion
- `file` (string) name of the test script to be created, including `.ts` extension. E.g. `my-test-script.ts`.
```

File: packages/docs/versioned_docs/version-1.0/guides/CLI.md [hunk 2]
```diff
@@ -35,12 +38,24 @@ Init a test script, a default configuration file and a minimal environment to ge
 element generate <file>
 ```
 
+**Positionals**
+
+- `file` (string) the test script file name to create. Specify a TypeScript file format, for example, `my-test-script.ts`.
```
Comment (HieuLeKMS) [inline packages/docs/versioned_docs/version-1.0/guides/CLI.md:43]: See my comment above

File: packages/docs/versioned_docs/version-1.3/guides/CLI.md [hunk 3]
```diff
@@ -36,12 +39,24 @@ Init a test script, a default configuration file and a minimal environment to ge
 element generate <file>
 ```
 
+**Positionals**
+
+- `file` (string) the test script file name to create. Specify a TypeScript file format, for example, `my-test-script.ts`.
```
Comment (HieuLeKMS) [inline packages/docs/versioned_docs/version-1.3/guides/CLI.md:44]: See my comment above

File: packages/docs/versioned_docs/version-1.5/guides/CLI.md [hunk 4]
```diff
@@ -24,24 +24,39 @@ element init [dir] [options]
...
 
 ### Generate a basic test script from a template
 
 ```shell
 element generate <file>
 ```
 
+**Positionals**
+
+- `file` (string) the test script file name to create. Specify a TypeScript file format, for example, `my-test-script.ts`.
```
Comment (HieuLeKMS) [inline packages/docs/versioned_docs/version-1.5/guides/CLI.md:44]: See my comment above

File: packages/docs/versioned_docs/version-2.0/guides/CLI.md [hunk 5]
```diff
@@ -38,12 +39,22 @@ Init a test script, a default configuration file and a minimal environment to ge
 element generate <file>
 ```
 
+**Positionals**
+
+- `file` (string) the test script file name to create. Specify a TypeScript file format, for example, `my-test-script.ts`.
```
Comment (HieuLeKMS) [inline packages/docs/versioned_docs/version-2.0/guides/CLI.md:44]: ☝️

File: packages/docs/docs/guides/CLI.md (hunk 1 above)
Comment (terryytran) [reply packages/docs/docs/guides/CLI.md:44]: I'm trying to write similar content to which Son wrote for `element run`
> - `file` (string) the test script (or path to the test script) to run. Specify a test script written in TypeScript with a `.ts` extension.

I do agree that "Specify a TypeScript file format" doesn't sound natural, so I'd to edit it as below. Let me know if you're OK with that
`- `file` (string) the test script name to generate. Specify a file name with a `.ts` extension (TypeScript), for example, `my-test-script.ts`.`

Comment (HieuLeKMS) [reply packages/docs/docs/guides/CLI.md]: Ok, I think that's better. Let's go with that

**Pull request merged**